LoopKit/TherapySetting.swift

```diff
@@ -0,0 +1,74 @@
+        case .suspendThreshold:
+            return LocalizedString("Suspend Threshold", comment: "Title text for suspend threshold")
+        case .basalRate:
+            return LocalizedString("Basal Rate", comment: "Title text for basal rate")
+        case .deliveryLimits:
+            return LocalizedString("Delivery Limits", comment: "Title text for delivery limits")
+        case .insulinModel:
+            return LocalizedString("Insulin Model", comment: "Title text for insulin model")
+        case .carbRatio:
+            return LocalizedString("Carb Ratio", comment: "Title text for carb ratio")
+        case .insulinSensitivity:
+            return LocalizedString("Insulin Sensitivities", comment: "Title text for insulin sensitivity")
+        case .none:
+            return ""
+        }
+    }
+    
+    var descriptiveText: String {
+        switch self {
+        case .glucoseTargetRange:
+            return LocalizedString("Correction range is the glucose value (or range of values) that you want Tidepool Loop to aim for in adjusting your basal insulin.", comment: "Descriptive text for glucose target range")
+        case .correctionRangeOverrides:
+            return LocalizedString("Pre-Meal and Workout ranges temporarily adjust your glucose target based on your selected activity.", comment: "Descriptive text for correction range overrides")
+        case .suspendThreshold:
+            return LocalizedString("When your glucose is predicted to go below this value, the app will recommend a basal rate of 0 U and will not recommend a bolus.", comment: "Descriptive text for suspend threshold")
+        case .basalRate:
+            return LocalizedString("Your basal rate of insulin is the number of units per hour that you want to use to cover your background insulin needs.", comment: "Descriptive text for basal rate")
+        case .deliveryLimits:
+            return LocalizedString("""
+                Maximum basal rate is the highest temporary basal rate Tidepool Loop is allowed to set automatically.
+
+                Maximum bolus is the highest bolus amount you can deliver at one time to cover carbs or bring down high glucose.
+            """, comment: "Descriptive text for delivery limits")
+        case .insulinModel:
+            return LocalizedString("The app assumes insulin is actively working for 6 hours. You can choose from two different models for how the app measures the insulin’s peak activity.", comment: "Descriptive text for insulin model")
+        case .carbRatio:
+            return LocalizedString("Your carb ratio is the number of grams of carbohydrate covered by one unit of insulin.", comment: "Descriptive text for carb ratio")
+        case .insulinSensitivity:
+            return LocalizedString("Your insulin sensitivities refer to the mg/dL drop in glucose expected from one unit of insulin.", comment: "Descriptive text for insulin sensitivity")
```

Comment:
This description isn't appropriate when the user's preferred glucose unit is mmol/L; we either need additional context to generate this string, or (preferable IMO) we can drop the glucose unit mention entirely.

Reply:
I'll add the above comment to IFU-24